delivery/src/main/java/nl/softwarestrijders/waiter/delivery/infrastructure/driven/storage/HttpCustomerRepository.java

```diff
@@ -0,0 +1,26 @@
+package nl.softwarestrijders.waiter.delivery.infrastructure.driven.storage;
+
+import nl.softwarestrijders.waiter.delivery.core.port.storage.CustomerRepository;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URI;
+import java.util.Objects;
+import java.util.UUID;
+
+public class HttpCustomerRepository implements CustomerRepository {
+    private final String rootPath;
+    private final RestTemplate client;
+
+    public HttpCustomerRepository(String rootPath, RestTemplate client) {
+        this.rootPath = rootPath;
+        this.client = client;
+    }
+
+    //Need to discuss with Milan what call to make to get the Customer and eventually the CustomerDeliveryAddress.
+    @Override
+    public CustomerDeliveryAddress getCustomerDeliveryAddress(UUID orderId) {
+        var uri = URI.create(this.rootPath + "/customer/order?order=" + orderId);
```

Comment:
maybe
```suggestion
        var uri = URI.create(String.format("%s/customer/order?order=%s", this.roothPath, orderId);
```

also, shouldn't it be `rootpath/customer/order/uuid`?